src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    public abstract String onScreen();
```

Comment:
this class should have common state (fields), for example you can store color here

Reply:
Changed